src/main/scala/com/wavesplatform/http/DebugApiRoute.scala

```diff
@@ -192,11 +194,15 @@ case class DebugApiRoute(ws: WavesSettings,
     ))
   def rollback: Route = (path("rollback") & post & withAuth) {
     json[RollbackParams] { params =>
-      ng.blockAt(params.rollbackTo) match {
-        case Some(block) =>
-          rollbackToBlock(block.uniqueId, params.returnTransactionsToUtx)
-        case None =>
-          (StatusCodes.BadRequest, "Block at height not found")
+      if (ng.height - params.rollbackTo > 1990)
```

Comment:
can we use the `2000` number from state here? 
in case that will be changed later